core/src/main/scala/com/rackspace/com/papi/components/checker/AffectsChecker.scala

```diff
@@ -0,0 +1,25 @@
+/***
```

Comment:
Hmm... I wanted to make AffectsChecker only usable within config. Splitting this out exposes the fact that that's not going to happen. Maybe make this private to config??  put it in the same place as config?